fire-hose-management-frontend/src/components/Departments/DepartmentsDataGrid.jsx

```diff
@@ -28,67 +27,70 @@ export const DepartmentsDataGrid = ({
                                     }) => {
     const classes = useStyles();
 
-    const buttons = [
-        {
-            key: 1,
-            labelText: "Abteilung löschen",
-            icon: <DeleteIcon/>,
-            onClick: () => {
-                setDeleteOverlayVisibleFunction(true)
-            }
-        }, {
-            key: 1,
-            labelText: "Abteilung bearbeiten",
-            icon: <EditIcon/>,
-            onClick: () => {
-                setIsEditOverlayVisibleFunction(true)
-            }
-        }]
-
-
-    const buttonHandler = (props) => {
-        if (props) {
-            return (
-                <div>
-                    {
-                        buttons.map((btn) => {
-                            return (
-                                <IconButton
-                                    aria-label={btn.labelText}
-                                    onClick={(value) => {
-                                        btn.onClick(value)
-                                    }}>
-                                    {btn.icon}
-                                </IconButton>
-                            )
-
-                        })
-                    }
-                </div>
-            )
-        }
+    const getFullName = (params) => {
```

Comment:
Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/StitZle/fire-hose-complete/pullRequest?prid=9542260)
```suggestion
  const getFullName = (params) => {
```